servers/src/main/java/tachyon/master/file/meta/InodeDirectory.java

```diff
@@ -177,9 +177,36 @@ public String toString() {
     return sb.toString();
   }
 
+  /**
+   * Converts the entry to an {@code InodeDirectory}.
```

Comment:
@jsimsa We had a short discussion about point three mentioned in #2102 and are going to change it towards using "**@link**" (see #2131).